test/ClangDisplayContext/test.cpp

```diff
@@ -0,0 +1,142 @@
+// This test is line- and column-sensitive, so test commands are at the bottom.
+// Tests are identical with Analysis/bug_hash_test.cpp
```

Comment:
They are identical today but will they have to be identical if `tests/Analysis/bug_hash_test.cpp` is changed/extended, etc.? If so, I would like to see a "forward link" in the other file to this file too... It's madness if "copied" files are only linked in one direction.